subprojects/core/src/main/java/org/gradle/api/Project.java

```diff
@@ -784,6 +802,31 @@
     ConfigurableFileTree fileTree(Object baseDir, Closure configureClosure);
 
     /**
+     * <p>Creates a new {@code ConfigurableFileTree} using the given base directory. The given baseDir path is evaluated
+     * as per {@link #file(Object)}. The action will be used to configure the new file tree. Example:</p>
+     *
+     * <pre autoTested=''>
+     * def myTree = fileTree('src') {
+     *    exclude '**&#47;.data/**'
```

Comment:
Although the syntax is correct and that's what we expect build authors using Groovy to write, this example actually represents a call to the `Closure` version of this method.

I think that's ok for now but we should consider reviewing the javadoc samples on `Action<T>` overloads in the future and use Kotlin syntax on those.